.env.development

```diff
@@ -1,2 +1,8 @@
 AUTH_SECRET=secret
-REDIRECT_URI=http://localhost:3000/auth-redirect
+DB_PORT=5432
+DB_USER=postgres
+DB_HOST=localhost
+DB_NAME=dev-directory
```

Comment:
The DB_STRING environment variable was split up into different pieces to make configuration easier (e.g. for tests, we use all the same values except `DB_NAME=test`.  The `DB_PASSWORD` is the only thing not in here because it's the only thing potentially sensitive.

Reply:
This is a great change, would you recommend we use this convention in Together / Binary Upload Zoom as well?